raiden/app.py

```diff
@@ -70,18 +70,21 @@ def __init__(
         default_secret_registry: SecretRegistry,
         default_service_registry: typing.Optional[ServiceRegistry],
         default_one_to_n_address: typing.Optional[Address],
+        default_msc_address: Address,
         transport,
         raiden_event_handler,
         message_handler,
         user_deposit: UserDeposit = None,
     ):
+        default_msc_address = default_msc_address
```

Comment:
what is this for?

Reply:
That was `default_msc_address = default_monitoring_service_contracts_address` to make the assigment in the following lines fit inside 100 lines. But then I had the same problem in many cases and decided to switch to the shorter name in all place, thus making this line nonsense. Removed!